### DISCUSSION TITLE
How to get MY ID on Rustdesk App for Android?

### DISCUSSION THREAD

**Discussion by pstein** [rustdesk/rustdesk#9413] [General]
When I start Rustdesk on Windows then I can see MY ID and I can enter a remote ID.

Now I want to connect from Windows to Android Rustdesk.
In order to find out the ID of Rustdesk App on my Andropid smartphone I started the Rustdesk App on Android.....but I cannot find MY ID.
Where is it?

**Comment by xlionjuan** [comment]
Start Service to get it

**Comment by pstein** [comment]
How do I start the service on Android?
Shouldn't it be started automatically when launching the App?

**Reply by xlionjuan** [reply]
Screen sharing.


No. It shouldn't

**Reply by pstein** [reply]
ok. Thank you

**Reply by VenusGirl** [reply]
Launch Rustdesk on your smartphone and accept the phone access permission request.

Select Screen Sharing from the Rustdesk bottom menu on your smartphone, press Start Service, and agree. Your ID and one-time password will be displayed.
